modules/virtualization/microvm/guivm.nix

```diff
@@ -57,6 +57,8 @@
             pkgs.waypipe
             pkgs.networkmanagerapplet
             pkgs.nm-launcher
+            pkgs.pamixer
+            pkgs.tcp-ssh
```

Comment:
We don't necessarily need to circle this stuff through overlay. You could also say `pkgs.callPackage ../../packages/tcp-ssh {}` here.

Reply:
Fixed